bugwarrior/db.py

```diff
@@ -89,7 +95,7 @@ def hamdist(str1, str2):
 
 def get_managed_task_uuids(tw, key_list, legacy_matching):
     expected_task_ids = set([])
-    for keys in key_list.values():
+    for keys in list(key_list.values()):
```

Comment:
Fair enough. I'm opposed to converting to `list` in cases where duck typing meets our need. If you disagree we can request comments from other contributors.